### PR TITLE
make queued build page render + cancel queued builds

### PR DESCRIPTION
**Pull request by zdhickman**

### CODE REVIEW

File: BlazarUI/app/scripts/components/repo-build/RepoBuildCancelButton.jsx [hunk 1]
```diff
@@ -27,7 +28,10 @@ class CancelBuildButton extends Component {
     this.setState({
       cancelling: true,
       showModal: false
-    });    
+    });
+
+    const parentPath = window.location.pathname.slice(0, window.location.pathname.lastIndexOf('/'));
```
Comment (zdhickman) [inline BlazarUI/app/scripts/components/repo-build/RepoBuildCancelButton.jsx]: @benrlodge any idea how to accomplish this without `window.location.pathname`? I'm trying to get back to the parent component - the branch build history page

Comment (benrlodge) [inline BlazarUI/app/scripts/components/repo-build/RepoBuildCancelButton.jsx]: use [browserHistory](https://github.com/reactjs/react-router/blob/master/docs/guides/Histories.md)

```
import { browserHistory } from 'react-router'
```

then you can trigger - 

```
browserHistory.goBack();
```

Comment (benrlodge) [inline BlazarUI/app/scripts/components/repo-build/RepoBuildCancelButton.jsx]: you can also do it one better and create a custom reusable history util module to store these types of functions, and import it where you need it.

Comment (benrlodge) [inline BlazarUI/app/scripts/components/repo-build/RepoBuildCancelButton.jsx]: On second thought, this assumes the user came from the browser history page.

Comment (benrlodge) [inline BlazarUI/app/scripts/components/repo-build/RepoBuildCancelButton.jsx]: Just pass the `this.props.params` prop down to the build page (starting from the container) and you can build the route from that.

**Pull request merged**